baseline.gradle

```diff
@@ -62,6 +62,21 @@ subprojects {
         removeUnusedImports()
         licenseHeaderFile "$rootDir/.baseline/copyright/copyright-header-java.txt"
       }
+
+      // Configure different scalafmt rules for specific Scala version
+      if (project.name.startsWith("iceberg-spark") && project.name.endsWith("2.13")) {
```

Comment:
Do we need to check `iceberg-spark`? It only affects the Scala code, right? Also, should we get the `scalaVersion` from the `System.getProperty` rather than from the project name?